python-server/README.md

```diff
@@ -0,0 +1,11 @@
+## Three Python Application using Three different frameworks
+ 
+### All Three script will create a python web server that listens on port 8989 and return "Hello World!" string when hit the endpoint below:
+
+### http://localhost:8989/landing
+
+
+To run the WepApp using different frameworks, run the command:
+ - Flask:  ```py py-flask.py```
+ - FastAPI:  ```py py-fastapi.py```
```

Comment:
provide instructions on how  to install dependencies prior to running these script